unity/Hello_World/Assets/RosMessages/UnityRoboticsDemo/msg/PosRotMsg.cs

```diff
@@ -0,0 +1,92 @@
+//Do not edit! This file was generated by Unity-ROS MessageGeneration.
```

Comment:
We could possibly clear out the `UnityRoboticsDemo` stuff if if we are not using any of these components.